src/lib/annotations/Annotator.scss

```diff
@@ -550,6 +548,22 @@ $tablet: 'max-width: 768px';
     }
 }
 
+.bp-annotation-draw .textLayer {
+    pointer-events: none;
+
+    > div {
+        pointer-events: auto;
+    }
+}
+
+.bp-annotation-layer-draw,
```

Comment:
@pramodsum @JustinHoldstock There's a little duplication with some highlight CSS, but it makes more sense to me since there are separate sections for annotation types and we didn't want the cursor to be text while drawing.